Makefile

```diff
@@ -69,7 +69,4 @@ cover: unit
 e2e: generate
 	./hack/e2e.sh
 
-e2e-prod:
```

Comment:
This change will break [how we run the e2e test in production](https://github.com/openshift/release/blob/b45cf4ff09658477410e14762284295b7338224f/ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml#L200); do you have any branch to fix this in the release repo?